src/electron/resources/license.txt

```diff
@@ -0,0 +1,12 @@
+Accessibility Insights for Android
```

Comment:
do we need both, `license.txt` and `license_en.txt`?

Reply:
Yep, for mac it will use license_en.txt, for linux it uses license.txt, for windows either will work